docs/source/in5min.rst

```diff
@@ -0,0 +1,124 @@
+.. _in5min:
+
+PIConGPU in 5 Minutes
+=====================
+
+A guide to run, but not understand PIConGPU.
+It is aimed at existing and preconfigured high performance computing clusters (“supercomputers”, also: HPC systems).
+For more information on the individual steps please refer to the manual.
+
+This guide needs **shell access** (probably via ``ssh``) and **some auxilliary programs** (e.g. ``git``).
+Consider getting familiar with the shell and git.
+Please also read the tutorial for your local HPC cluster.
+
+We will use the following directories:
+
+- ``~/src/picongpu``: source files from github
+- ``~/picongpu.profile``: load the dependencies for your local environment
+- ``~/picongpu-projects``: scenarios to simulate
+- ``/scratch/mydir``: result data of the simulation runs
+
+Please replace them whenever appropriate.
+
+Get the source
+--------------
+
+Use git to obtain the source and use the current ``dev`` branch and put it into ``~/src/picongpu``::
+
+  mkdir -p ~/src
+  git clone -b dev https://github.com/ComputationalRadiationPhysics/picongpu ~/src/picongpu
```

Comment:
By default, the `master` branch will be checked out, but above you are speaking about the `dev` branch. Perhaps either stating to use the `master` branch or adding `git checkout dev # switching to the dev brach` would help here.

Reply:
I've gone for `-b dev` to check out `dev` in the `git clone` call, but this might be confusing for/overlooked by novices.
-> Will add explicit line w/ `git checkout`

Reply:
Ahh sorry 🤦 - didn't see the `-b dev` flag - it was now an unintentional DAU test from my side 😓 -  I think your solution to separate the steps is a very good way to go :+1:

Reply:
addressed in 7b1164f2